python/ray/train/v2/tests/test_data_parallel_trainer.py

```diff
@@ -10,10 +10,13 @@
 from ray.train.backend import Backend
 from ray.train.constants import RAY_CHDIR_TO_TRIAL_DIR, _get_ray_train_session_dir
 from ray.train.tests.util import create_dict_checkpoint
+from ray.train.v2._internal.constants import is_v2_enabled
 from ray.train.v2._internal.exceptions import TrainingFailedError
 from ray.train.v2.api.data_parallel_trainer import DataParallelTrainer
 from ray.train.v2.api.result import Result
 
+assert is_v2_enabled()
+
```

Comment:
Hmm is this expected to be verified globally? Would it make sense to move to `__main__` or as a mini test of its own?

e.g.
```python
def test_v2_enabled():
    assert is_v2_enabled()
```

Reply:
I'll remove this. Mostly was for me to test locally and make sure to not forget the env var.